src/backend/core/api/__init__.py

```diff
@@ -16,18 +17,10 @@ def exception_handler(exc, context):
     This code comes from twidi's gist:
     https://gist.github.com/twidi/9d55486c36b6a51bdcb05ce3a763e79f
     """
-    if isinstance(exc, ValidationError):
-        detail = None
-        if hasattr(exc, "message_dict"):
-            detail = exc.message_dict
-        elif hasattr(exc, "message"):
-            detail = exc.message
-        elif hasattr(exc, "messages"):
-            detail = exc.messages
-
-        exc = drf_exceptions.ValidationError(detail=detail)
-
-    return drf_views.exception_handler(exc, context)
+    if isinstance(exc, DjangoValidationError):
+        exc = drf_exceptions.ValidationError(as_serializer_error(exc))
+
+    return drf_exception_handler(exc, context)
```

Comment:
Did you try to remove the usage of our own exception handler and use directly the one from `drf_standardized_errors` on the `REST_FRAMEWORK` settings ?

Reply:
I just tried and it doesn't catch django core ValidationError:

```python
  File "/app/core/models.py", line 930, in move
    raise ValidationError(
django.core.exceptions.ValidationError: {'target': ['Only folders can be targeted when moving an item']}
INFO     request.summary:middleware.py:82
```